auth/handler.go

```diff
@@ -228,27 +228,18 @@ func (h *AuthenticationHandler) IsAuthenticated(req *http.Request) (bool, *JWTRe
 	} else if !ok {
 		valid, claims, err := h.verifier.VerifyToken(token.JWT)
 		if err == nil && valid {
-			exp, ok := claims["exp"]
+			stdClaims, ok := claims.(jwt.StandardClaims)
 			if !ok {
-				h.expLock.Lock()
-				h.expCache[token.JWT] = 0
-				h.expLock.Unlock()
-				return true, token, nil
-			}
-
-			expNum, ok := exp.(float64)
-			if !ok {
-				return false, nil, fmt.Errorf("expiration tstamp is not numeric")
+				return false, nil, fmt.Errorf("error while casting claims")
 			}
 
-			expNumInt := int64(expNum)
-			if expNumInt > time.Now().Unix() {
-				h.logger.Debugf("JWT for token %s expires at %d", token, expNumInt)
+			if stdClaims.ExpiresAt > time.Now().Unix() {
```

Comment:
Shouldn't `h.expCache[token.JWT]` still be set to `0` if no expiration is set at all?